litmus-portal/frontend/src/components/CustomCard/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import CardContainer from './CardContainer';
+import CardContent from './CardContent';
+import { preDefinedWorkflowData } from '../../models/predefinedWorkflow';
+
+const CustomCard: React.FC<preDefinedWorkflowData> = ({ ...props }) => {
+  return (
+    <CardContainer>
+      <CardContent {...props} />
```

Comment:
props won't be allowed by linter, it would tell you to destructure it

Reply:
Actually its not throwing any error.